### ISSUE TITLE
Wulfrik's Wyrmpiercer ammo upgrade affects Mammoths/doesn't affect RoR mammoth

### ISSUE THREAD

**Issue by Harmandodelavega** [chadvandy/cbfm_wh3#536]
Don't know which would be the bug

**Comment by floskan** [comment]
Wyrmpiercer worked for mammoths in 2 when they revamped his personal skill line, so i guess intended?

**Comment by Harmandodelavega** [comment]
It might have not been intended in 2 as well

**Comment by floskan** [comment]
I just thought it made sense since it's spear-chucking marauders on top of the mammoth.

Wasn't really seen as a bug back then: https://www.reddit.com/r/totalwar/comments/qk20ez/does_wulfricks_juggernauts_of_the_sarl_applied_to/

**Comment by Harmandodelavega** [comment]
Yeah, it's just weird I don't think it's intended but if it is the RoR is bugged

**Comment by Harmandodelavega** [comment]
#420 #428
